### ISSUE TITLE
How to evaluate the confidence of a multimer ？

### ISSUE THREAD

**Issue by genomicsgroup** [facebookresearch/esm#484]
Dear developer. Thank you for ESM and ESMFold!
  The AlphaFold-multimer program generate the iptm & ptm score to evaluate the confidence of prediction multimers，but ESMFold only generate pLDDT & pTM score in stdout, how to evaluate the confidence of ESMFold multimers？

**Comment by genomicsgroup** [comment]
Is there any other way to calculate the iptm or dockQ/pdockQ value for multimer? Your opinions are very important to me!

**Comment by nikitos9000** [comment]
Yes it's possible to calculate DockQ score, please follow https://github.com/bjornwallner/DockQ/ – it only requires ESMFold .pdb output to compute this score against natural structure.
